docs/examples.md

```diff
@@ -134,6 +147,36 @@ rule "Humidity: Control ExhaustFan" do
 end
 ```
 
+## Executing an External Command
+
+OpenHAB offers {OpenHAB::Core::Actions::Exec.execute_command_line execute_command_line} to execute
+an external command. However, in Ruby it is also possible to use, amongst others:
+
+- [system](https://ruby-doc.org/core/Kernel.html#method-i-system): waits for the execution to finish,
+  then returns `true` (zero exit status), `false` (non-zero exit status), or `nil` if the execution fails.
+- [backtick operator](https://ruby-doc.org/core/Kernel.html#method-i-60): waits for the execution to finish,
+  then returns the stdout output of the command as a String.
+- [spawn](https://ruby-doc.org/core/Kernel.html#method-i-spawn): executes the command and immediately
+  returns control to the caller, leaving the command running in the background. This method returns
+  the pid of the spawned process which must be waited out or detached to avoid creating zombie processes.
+- [IO#popen](https://ruby-doc.org/core/IO.html#method-c-popen): a more advanced method of executing an external process.
+
+Unlike {OpenHAB::Core::Actions::Exec.execute_command_line execute_command_line} which expects each command
+argument to be split up, Ruby's execution methods only expects a single string containing the full command.
```

Comment:
This isn't quite accurate. While Ruby's methods can be used this way, it also allows the openHAB way of specifying the command and arguments separately (which is actually safer and doesn't need escaping). It even allows using an array for the command, where the first element is used to set the command, and the second element becomes argv0 in the new process (i.e. the command used to execute the program; how the shell sets it when using a symlink, or when it's on the path, etc.).